rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -377,33 +377,31 @@ namespace rocal{
             .def(
                 "at",
                 [](rocalTensor &output_tensor, uint idx) {
-                    const auto& info = output_tensor.info();
-                    std::vector<size_t> stride_per_sample(info.strides());
+                    std::vector<size_t> stride_per_sample(output_tensor.strides());
                     stride_per_sample.erase(stride_per_sample.begin());
-                    std::vector<size_t> dims(info.dims());
+                    std::vector<size_t> dims(output_tensor.dims());
                     dims.erase(dims.begin());
                     py::array numpy_array;
-                    switch (output_tensor.info().data_type())
+                    switch (output_tensor.data_type())
                     {
-                    case RocalTensorDataType::UINT8:
-
+                    case RocalTensorOutputType::ROCAL_UINT8:
                         numpy_array = py::array(py::buffer_info(
-                            ((unsigned char *)(output_tensor.buffer())) + idx * (info.strides()[0]/sizeof(float)),
+                            ((unsigned char *)(output_tensor.buffer())) + idx * (output_tensor.strides()[0]/sizeof(float)),
```

Comment:
why is this dividing by sizeof(float) for UINT8

Reply:
Fixed